### PR TITLE
style: Add rounded borders on original comment box

### PR DESCRIPTION
**Pull request by RebootedDuck**
This adds "border-radius: 20px;" to the original comment window on new/old Reddit, this should improve view-ability and make the visual styling slightly more appealing

Sorry if this was an already considered suggestion that was rejected as I couldn't find anything

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -898,6 +898,7 @@
                         font-size: 14px;
                         padding: 16px;
                         line-height: 20px;
+                        border-radius: 20px;
```
Comment (DenverCoder1) [inline script.js:901]: image
image


```suggestion
                        border-radius: 4px;
```

File: script.js [hunk 2]
```diff
@@ -1040,6 +1041,7 @@
...
```
Comment (DenverCoder1) [inline script.js:1044]: image
image

```suggestion
                        border-radius: 7px;
```

**Review by DenverCoder1 — COMMENTED** [review]
Looks good, but I think maybe less rounded would be better. In my opinion, it should ideally match the roundness of the other elements on the page (`4px` on redesign, `7px` on old reddit)

Feel free to use the commit suggestions buttons if you agree and let me know what you think.

**Comment by RebootedDuck** [comment]
Oh great point, I didn't think to check what rounding other elements were using

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for the contribution :tada:

**Comment by RebootedDuck** [comment]
Cheers!

**Pull request merged**